### PR TITLE
Added content inside IDEs and VS Code sections, present inside DevRel Roadmap

### PR DESCRIPTION
**Pull request by SatyamVyas04**
- Added history and basics of IDEs
- Added VSCode guide and official links

### CODE REVIEW

**Comment by SatyamVyas04** [comment]
@dansholds 🙏🏻

File: src/data/roadmaps/devrel/content/vs-code@j5nNSYI8s-cH8EA6G1EWY.md [hunk 1]
```diff
@@ -1 +1,11 @@
-# VS Code
\ No newline at end of file
+# VS Code
+
+Visual Studio Code (VSCode) was first announced by Microsoft in 2015 and quickly became one of the most popular and widely used Integrated Development Environments (IDEs). Built on [Electron](https://www.electronjs.org/), a framework that allows web technologies like JavaScript, HTML, and CSS to create desktop applications, VSCode offers a lightweight and highly extensible platform for developers. VSCode focuses on being a streamlined code editor with the ability to install extensions that add features such as debugging, version control, and language-specific tooling. Microsoft's vision was to create a flexible environment that could cater to all types of developers, from beginners to seasoned professionals.
+
+Over time, VSCode has evolved with powerful built-in features like [IntelliSense](https://code.visualstudio.com/docs/editor/intellisense), an integrated terminal, [Git](https://git-scm.com/) support, and remote development capabilities. It supports numerous programming languages and can be customized with themes, plugins, and settings to fit a wide range of development workflows.
```
Comment (dansholds) [inline src/data/roadmaps/devrel/content/vs-code@j5nNSYI8s-cH8EA6G1EWY.md:5]:
```suggestion


```

File: src/data/roadmaps/devrel/content/vs-code@j5nNSYI8s-cH8EA6G1EWY.md [hunk 2]
```diff
@@ -1 +1,11 @@
-# VS Code
\ No newline at end of file
+# VS Code
+
+Visual Studio Code (VSCode) was first announced by Microsoft in 2015 and quickly became one of the most popular and widely used Integrated Development Environments (IDEs). Built on [Electron](https://www.electronjs.org/), a framework that allows web technologies like JavaScript, HTML, and CSS to create desktop applications, VSCode offers a lightweight and highly extensible platform for developers. VSCode focuses on being a streamlined code editor with the ability to install extensions that add features such as debugging, version control, and language-specific tooling. Microsoft's vision was to create a flexible environment that could cater to all types of developers, from beginners to seasoned professionals.
+
+
```
Comment (dansholds) [inline src/data/roadmaps/devrel/content/vs-code@j5nNSYI8s-cH8EA6G1EWY.md:5]:
```suggestion
```

File: src/data/roadmaps/devrel/content/ides@KdFYmj36M2jrGfsYkukpo.md [hunk 3]
```diff
@@ -1 +1,13 @@
-# IDEs
\ No newline at end of file
+# IDEs
+
+The concept of Integrated Development Environments (IDEs) traces its roots back to the 1960s and 1970s when basic text editors and simple programming tools were used in early computing. The first notable IDEs emerged with the rise of Unix systems in the 1970s, such as the EMACS text editor, which included features like code editing and debugging.
+
+In the 1980s, with the growth of personal computing, IDEs became more advanced. Turbo Pascal, introduced by Borland, set a new standard by offering a fast, compiled programming environment. Microsoft and Apple followed suit, integrating more sophisticated tools into their ecosystems. As development workflows became more complex, IDEs evolved to include debugging, testing, and source control capabilities.
+
+Today, IDEs have become essential for developers, supporting multiple programming languages and integrating cloud-based tools, continuous integration, and real-time collaboration. IDEs like Visual Studio Code, IntelliJ IDEA, and Eclipse are widely adopted by developers across industries.
```
Comment (dansholds) [inline src/data/roadmaps/devrel/content/ides@KdFYmj36M2jrGfsYkukpo.md:7]:
```suggestion
The concept of Integrated Development Environments (IDEs) traces its roots back to the 1960s and 1970s when basic text editors and simple programming tools were used in early computing. The first notable IDEs emerged with the rise of Unix systems in the 1970s, such as the EMACS text editor, which included features like code editing and debugging. Today, IDEs have become essential for developers, supporting multiple programming languages and integrating cloud-based tools, continuous integration, and real-time collaboration. IDEs like Visual Studio Code, IntelliJ IDEA, and Eclipse are widely adopted by developers across industries.
```

**Pull request merged**